### PR TITLE
Add admonitions support to website

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

**Comment by iranzo** [comment]
Related with #95 can you review @matthewcarleton if this is the proper way to include it?

File: _sass/_premonition.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    color: $default-color;
+  }
+
+  .fa {
+    font-size: 18px;
+    opacity: .3;
+    padding-top: 2px;
+    padding-left: 20px;
+  }
+
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
svg {
margin:13px 0 0 15px;
}
```
fixes the odd spacing for the icon

File: _layouts/default.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {% include head.html %}
 {% include header.html %}
+<script defer src="https://use.fontawesome.com/releases/v5.1.0/js/all.js" integrity="sha384-3LK/3kTpDE/Pkp8gTNp2gR/2gOiwQ6QaO7Td0zV76UFJVhqLl4Vl3KL1We6q6wR9" crossorigin="anonymous"></script>
```
Comment (matthewcarleton) [inline _layouts/default.html]: can you put this in the head.html include

Comment (iranzo) [reply _layouts/default.html]: Thanks, let me ammend it :)

**Comment by iranzo** [comment]
imagen


Much better

File: _sass/_premonition.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  $info-light-color: #f3f8f3;
+  $warning-color: #f0ad4e;
+  $warning-light-color: #fcf8f2;
+  $error-color: #d9534f;
+  $error-light-color: #fdf7f7;
+  $content-color: rgba(0,0,0,0.5);
+
+  display: grid;
+  grid-template-columns: 43px auto;
+  padding-top: 13px;
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
  padding-top: 6px;
```

File: _sass/_premonition.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  .fa {
+    font-size: 18px;
+    opacity: .3;
+    padding-top: 2px;
+    padding-left: 20px;
+  }
+svg {
+margin:13px 0 0 15px;
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
margin:11px 0 0 15px;
```

**Comment by iranzo** [comment]
I've added the same change to KV website: https://github.com/kubevirt/kubevirt.github.io/pull/348 the note there with 'title' looks a bit strange

**Comment by iranzo** [comment]
imagen

**Comment by iranzo** [comment]
Seems that GH doesn't show it:

imagen

**Comment by iranzo** [comment]
Reverting

**Pull request merged**